bundle-workflow/README.md

```diff
@@ -216,3 +217,17 @@ The following options are available.
 |--------------------|-------------------------------------------------------------------------|
 | --keep             | Do not delete the temporary working directory on both success or error. |
 | -v, --verbose      | Show more verbose output.                                               |
+
+### Onboarding Requirements
+
+With many components included in the distribution each component has the responsibility to keep the release process streamlined and react to integration issues.
+
+- Components repositories must have a tag matching all manifest versions so issue can be appropriately tagged.
+- Component maintainers must be able to respond to blocking issue within 2 business days.
```

Comment:
I would say "Component maintainers regularly triage their issues and respond to issues blocking the bundle workflow promptly.". If you're going to impose an SLA we need to discuss this with the said component owners first.

Reply:
Nixing this section out - feels heavy handed on re-read